README.md

```diff
@@ -2,179 +2,128 @@
-
-default path install is:
-> /usr/lib/node_modules/gtp2ogs/
-
-- For windows, open a node.js command prompt as admin, then run this command:
-
-```Shell
-npm install -g gtp2ogs
-```
-
-default path install is something like this:
-> C:\Users\yourusername\AppData\Roaming\npm\node_modules\gtp2ogs\
-
-On all operating systems, gtp2ogs will be installed in 2 different directories, but
-**the one that needs to be run with node is gtp2ogs.js in node_modules directory**
-
-### 3. Recommended: Upgrade to devel branch
-
-This step is **highly recommended**
-
-By default, npm installs an old branch of gtp2ogs that does not include latest
-improvements, new features, and fixes
-
-An easy way to upgrade is to copy all the devel gtp2ogs files and folders
-(bot.js, config.js, etc..) to the original directory where gtp2ogs is
-installed, and overwrite the old existing files
-
-note: before overwriting, you can backup your old files so that you can
-go back to the old branch of gtp2ogs anytime later if you want
-
-Then it is needed to do the post install:
+## Options
 
-The command below will automatically detect all missing packages needed
-from the package.json of the new branch, and install all these packages
+### Options-List
 
-- for linux:
+You can find a list of all possible to use gtp2ogs arguments [here](/docs/OPTIONS-LIST.md).
 
-```Shell
-cd /usr/lib/node_modules/gtp2ogs/
-sudo npm install
-```
+### Options settings
 
-- for windows:
+Before putting `<gtp2ogsarguments>`, you have to first put the mandatory options:
 
-Open a node.js command prompt as admin, then:
+- [--username](/docs/OPTIONS-LIST.md#username)
+- [--apikey](/docs/OPTIONS-LIST.md#apikey)
 
-```Shell
-pushd C:\Users\yourwindowsusername\AppData\Roaming\npm\node_modules\gtp2ogs\
-npm install
-```
+Then, choose the wanted `<gtp2ogsarguments>` from the above list of all options section
+ (no need to use all these options, only the ones you want), and separate each option with a space.
 
-For details or help, you can see:
+note: some gtp2ogsarguments have default so they are enabled even if you don't specify them.
 
-- for linux: [3A3) Recommended: Upgrade gtp2ogs.js from old branch to “devel” branch (latest)](https://github.com/wonderingabout/gtp2ogs-tutorial/blob/master/docs/3A3-linux-optional-upgrade-to-devel.md)
-- for windows: [3B3) Recommended: Upgrade gtp2ogs from old branch to devel (latest) branch](https://github.com/wonderingabout/gtp2ogs-tutorial/blob/master/docs/3B3-windows-optional-upgrade-to-devel.md)
+After that, it is mandatory to also yout AI settings. To do that, add a `--`
+ (with one space before and one space after the `--` ), to separate `<gtp2ogsarguments>`
+ from your bot path and `<botarguments>`.
 
-### 4. Most common usage: start gtp2ogs.js using nodejs
+## Beta OGS
 
-For linux (preferably as sudo):
+Before you can play on official [OGS server](https://online-go.com/) server, OGS admins
+ will often ask that you run tests to make sure everything is running fine.
```

Comment:
I did not find this to be true

Reply:
indeed they dont always ask it, but they often do, especially if they see the new bot admin is fairly new
so its indeed not always true but often